test/test_utils/__init__.py

```diff
@@ -321,7 +321,8 @@ def get_inference_run_command(image_uri, model_names, processor="cpu"):
     server_type = get_inference_server_type(image_uri)
     if processor == "eia":
         multi_model_location = {
-            "resnet-152-eia": "https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-152-eia.mar",
+            "resnet-152-eia": "https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-152-eia-1-7-0.mar",
+            "resnet-152-eia-1-5-1": "https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-152-eia-1-5-1.mar",
```

Comment:
Add logic to use model name as per the image version (v1.5.1 and others) for ECS [test](https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ecs/mxnet/inference/test_ecs_mxnet_inference.py#L38) and EC2 [test](https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py#L12)

Reply:
good point. Added the logic to decide `model_name` by framework version